### PR TITLE
Chore: Replace previous UI component package with Prefect UI Library

### PR DESCRIPTION
**Pull request by znicholasbrown**
Renames the UI directory, updates package refs to [Prefect UI Library](https://github.com/PrefectHQ/prefect-ui-library)

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ui-orion",
-  "version": "0.1.0",
+  "name": "@prefecthq/ui",
+  "version": "2.8.0",
```
Comment (pleek91) [inline ui/package.json:3]: Nice. Are we going to try and keep this in sync with the prefect version?

**Review by pleek91 — DISMISSED** [review]

File: ui/package.json (hunk 1 above)
Comment (znicholasbrown) [reply ui/package.json:3]: I'd love to if we could!

**Comment by znicholasbrown** [comment]
nice catch @pleek91 - I'll leave the other part of codeowners to the oSS team

**Comment by pleek91** [comment]
@znicholasbrown oh and the build test is failing cause of the folder change I assume

**Comment by znicholasbrown** [comment]
yeah working on the workflows now, completely forgot about those

**Comment by znicholasbrown** [comment]
This branch got messed up from the rebase, gonna open up a new one against the proper base

**Pull request closed without merging**